app/models/tag_selection.rb

```diff
@@ -19,4 +19,10 @@ def ruser
   def tagname
     tag.name
   end
+
+  def self.users_following_everything_tag
```

Comment:
I think you should be able to use `Tag.followers()` instead of a custom method here -- 

https://github.com/sagarpreet-chadha/plots2/blob/6106522b57a4bfe142edb181cf95741d6234ba1e/app/models/tag.rb#L131